### DISCUSSION TITLE
Now registrations are open for REAL-TIME STREAMING HACKATHON 🔔

### DISCUSSION THREAD

**Discussion by umtcylmz** [ant-media/Ant-Media-Server#5905] [Announcements]
## ℹ️ About the Challenge 
Hey developers, creative minds worldwide! Join our hackathon and create cool apps or plugins with Ant Media Server for a shot at winning $5,000 in prizes! 🚀

After signing up, you'll get a starter kit, and our Ant Media experts will be there to help you out during the hackathon.

Get creative! Make awesome streaming apps or plugins for mobile or web on topics like chatting, IoT, data, and gaming. The sky's the limit! Don't wait—[register now](https://antmedia.io/hackathon/)!

## 🏆 Prize Pool 
**🥇 1st prize: $2500 
🥈 2nd prize: $1500
🥉 3rd prize: $1000**

If you successfully finish the development process and create a product at the end, on [Ant Media marketplace](https://antmedia.io/marketplace/) you will have the opportunity to promote and also sell it **[Register Now!](https://antmedia.io/hackathon/)**

## 📆 Agenda

| Event | Date |
|---|---|
| ⏺ Challenge introduction  | 1st of December |
| 📨 Registration process starts |  5th of December |
| 📹  Live session – Pre-event |  18th of December 2023 at 18:00pm GMT+3 |
| 🚀 Hackathon period | 1st of December from 11:00am GMT+3 through 14th of January 15:00pm GMT+3 |
| 🛑 Hackathon Day # 1 Q&A with Mentors | 8th of January  18:00pm GMT+3 |
| 🛑 Project submission deadline | 14th of January  12:00pm GMT+3 |
| 📹 Demo presentations & judging period | 14th of January  14:00pm GMT+3 |
| 🏆 Award ceremony | 14th of January at 15:00pm GMT+3 |

Ant Media started organizing hackathons in [2022](https://antmedia.io/the-story-of-real-time-streaming-hackathon-2022/). Here is how those events went this year in [March](https://antmedia.io/what-happened-in-the-first-hackathon-of-the-year-ant-media-will-organize-3-more-hackathons-in-2023/) and [July](https://antmedia.io/3rd-real-time-streaming-hackathon/).

**Comment by mariia-artamonova** [comment]
Watch how our Hackathon pre-event went here ⚡ https://youtu.be/LjJo8iymWPI?feature=shared

Use timestamps to navigate between event parts easier. Get familiar with this hackathon’s challenge, helpful resources that you can use and project submission requirements.

Here is the [presentation](https://docs.google.com/presentation/d/1DrWSnRud-4SvRklmBiVUJ9GPCltBJsx4/edit?usp=sharing&ouid=100837637668379219714&rtpof=true&sd=true) we discussed.

Big thanks to everyone who joined us yesterday – special shout-outs to [rishitha madhavaram](https://www.linkedin.com/in/ACoAADrwEkwB12Q4QK9erkOxxFhliKXvlPRFAdY) and [Rahul kumar](https://www.linkedin.com/in/ACoAACmE9fwBxluhllG9MFxYg_MHrAOJrvTBiAg)! And of course thank you [Umut Can Yılmaz](https://github.com/umtcylmz) for hosting the event. It was truly a pleasure to connect with you all. Looking forward to more meaningful interactions in the future ❤️

What is next?
1. [Submit your project idea](https://forms.gle/FWeDrSns6ktQedZF8) to complete registration in this hackathon
2. [Save the next event date on your calendar](https://calendar.google.com/calendar/u/0/r/eventedit/copy/NHJhZWJwNWpqcjhjajI0Mjk5M21nbzRpdGUgY180NzkzYWYyOTlmYjIwZmEwOTJlNzEyYTNmMzdkNjBlYjg1YjE4ZmEzNTUyMDAxY2E0ODJhODNmZjAxNjMyZDYyQGc/bWFyaWlhLmFydGFtb25vdmFAYW50bWVkaWEuaW8)
